rclpy/test/test_create_node.py

```diff
@@ -43,7 +43,8 @@ def test_create_node_with_namespace(self):
     def test_create_node_with_empty_namespace(self):
         node_name = 'create_node_test'
         namespace = ''
-        node = rclpy.create_node(node_name, namespace=namespace, context=self.context)
+        node = rclpy.create_node(
+            node_name, namespace=namespace, context=self.context)
```

Comment:
@jubeira nit: why the change?

Reply:
Reverted in https://github.com/ros2/rclpy/pull/325/commits/4d2429d48296b76742448dccf74d8f4450c43ca9. This unwanted change had got through the rebase.